src/Servers/Kestrel/Core/src/KestrelServerOptions.cs

```diff
@@ -252,9 +252,6 @@ internal void Serialize(Utf8JsonWriter writer)
         writer.WritePropertyName(nameof(AllowResponseHeaderCompression));
         writer.WriteBooleanValue(AllowResponseHeaderCompression);
 
-        writer.WritePropertyName(nameof(EnableAltSvc));
```

Comment:
@davidfowl @Tratcher `EnableAltSvc` is marked as `obsolete` and causing build errors. I am removing from here. since the message explicitly say that it no longer has any impact on runtime behavior, that will cause the property not to be serialized when emitting the kestrel (`Microsoft-AspNetCore-Server-Kestrel`) configuration event (`ID: 11`).

Reply:
👍